docs/plugins/image.md

```diff
@@ -1,77 +1,81 @@
+It is recommended that you *optimize* the local assets of your App by resizing them to the *exact* size and quality in which you will use them. This positively affects the memory usage of your App.
 
-However when you don't have control over the images you're displaying in your App (because they come from a remote API for example), you can use the Image plugin to resize and crop images.
+However, if you don't have control over the images to be displayed in your App (for example, because they originate from a remote API), you can use the *Image* plugin to resize and crop them.
 
 ## Usage
 
-Whenever you need to resize image, import the Image plugin from the Lightning SDK
+Import the Image plugin from the Lightning SDK in components where you want to resize an image:
 
 ```js
 import { Img } from '@lightningjs/sdk'
 ```
 
-## Available methods
+## Available Methods
 
-### Exact
+### exact()
 
-Resizes the image to the exact dimensions, ignorning the ratio.
+
+Resizes the image to the exact dimensions, ignoring the ratio.
 
 ```js
 Img(url).exact(width, height)
 ```
 
-### Landscape
+### landscape()
 
 Resizes the image by width, maintaining the ratio.
 
 ```js
 Img(url).landscape(width)
 ```
 
-### Portrait
+### portrait()
 
-Resizes the image by height, maintaining the ratio
+Resizes the image by height, maintaining the ratio.
 
 ```js
 Img(url).portrait(height)
 ```
 
-### Cover
+### cover()
 
-Resizes the image in such a way that it covers the entire area. Depending on the orientation (portrait or landscape) of the image it will resize the image by width or by height.
+Resizes the image in such a way that it covers the entire area. Depending on the orientation (portrait or landscape) of the image and the target, it resizes the image by width or by height.
 
 ```js
 Img(url).cover(width, height)
 ```
 
-### Contain
+### contain()
 
-Resizes the image in such a way that it is contained within the available area. Depending on the orientation (portrait or landscape) of the image it will resize the image by width or by height.
+Resizes the image in such a way that it is contained within the available area. Depending on the orientation (portrait or landscape) of the image and the target, it resizes the image by width or by height.
 
 ```js
 Img(url).contain(width, height)
 ```
 
-### Original
+### original()
 
-Generate an image without resizing it (i.e. use the original dimensions), while still passing it through the proxy (and taking advantage of caching).
+Generates the image without resizing it (that is, it uses the original dimensions), while still passing it through the proxy (and taking advantage of caching).
 
 ```js
 Img(url).original()
 ```
 
-## Image quality
+## Image Quality
+
+To increase the performance on lower-end boxes –especially those with limited GPU memory– there is a platform setting `image.quality` which enables you to control the *image quality*.
```

Comment:
I think platform should be stressed (italic), to indicate it's not an application setting.

Reply:
I'm not sure if the italic emphasis indicates that it's not an application setting. Isn't there another way of pointing this out? E.g., "[...] there is an _external_ platform setting 'image.quality' which [...]"?

Reply:
I think using "external" could add more confusion actually (cause we don't mention that anywhere else). The importance is to distinguish that image quality is placed in the 'platform' section of the Settings. Perhaps linking to the appropriate docs on settings will be enough?

Reply:
I'll include a link to [http://www.lightningjs.io/docs/#/lightning-sdk-reference/plugins/settings?id=settings](http://www.lightningjs.io/docs/#/lightning-sdk-reference/plugins/settings?id=settings), where the Application and Platform Settings stored in **settings.json** are introduced.